src/Sitegen/Models/Config/Config.cs

```diff
@@ -8,7 +8,11 @@ public class Config : IDeserialized
         public string LayoutsDir { get; set; }
         public string OutputDir { get; set; }
         public string PostsDir { get; set; }
-        public LineBreaks? LineBreaks { get; set; }
+
+        // Enabling "soft line breaks as hard" is currently the default. Can be opted out by individual blog posts
+        // as needed.
+        public LineBreaks LineBreaks { get; set; } = Models.LineBreaks.Hard;
+
```

Comment:
I inlined the default value here, making it possible to make this field non-nullable. :tada: